prototype.py

```diff
@@ -0,0 +1,34 @@
+import random
+#modified code from introduction to computing and programming in python a multimedia approach,
+#third edition, mark j. Guzdial,Barbara Ericson,p.~149
+#Replaces a colour in an image with another image preferably of the same size
+def greenScreen(forGrnd,replaceColour,threshold,backGrnd):
```

Comment:
I'm assuming forGrnd is supposed to be foreGrnd. Its role may be clearer if spelling is corrected. Also, omitting vowels in variable names (i.e. Grnd instead of Ground) makes it slightly less readable and clear.